Dockerfile

```diff
@@ -1,54 +1,65 @@
-FROM debian:jessie
+FROM docker.elastic.co/elasticsearch/elasticsearch:5.3.0
```

Comment:
We could use `FROM docker.elastic.co/elasticsearch/elasticsearch-alpine-base:latest` but after that I imagine we could just copy in the contents of https://github.com/elastic/elasticsearch-docker/blob/master/build/elasticsearch/Dockerfile into this directory and remove the bits we don't want. We'd need to put the LICENSE notice for their repo at the top of that file.